### PR TITLE
feat: added contentful pages and menu

### PR DESCRIPTION
**Pull request by owenmerry**

### CODE REVIEW

File: packages/website/src/app.jsx [hunk 1]
```diff
@@ -122,12 +113,41 @@ const App = () => {
...
+                      'https://docs.google.com/forms/d/e/1FAIpQLSc9AFF9oQI2bDKWf9heplIi3hPQE6Ok2v-zGObByD9T0EjlbQ/viewform'
+                    )}
+                />
 
+                <Switch>
+                  {/* static pages */}
+                  {/* <Route exact path="/" component={Dynamic} /> */}
+                  {/* <Route path="/club" component={Dynamic} /> */}
+                  {/* <Route path="/ethos" component={Ethos} /> */}
+                  {/* <Route path="/diversity" component={Diversity} /> */}
```
Comment (owenmerry) [inline packages/website/src/app.jsx:137]: I can clean these away as not needed actually

File: packages/website/src/shared/contentful.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import contentful from 'contentful'
+
+export const setupClient = () => {
+  const client = contentful.createClient({
+    space: '6dcasuqyhf2a',
```
Comment (owenmerry) [inline packages/website/src/shared/contentful.js:5]: These are exposed during api calls so are not that big an issue been in the code, I can look at being able to use env files with our setup but can then replace these codes after

Comment (achingbrain) [reply packages/website/src/shared/contentful.js:5]: Yeah, I think better to access these things through config.

We follow this pattern elsewhere - for example our stripe publishable key gets [added to the config as it's sandbox version](https://github.com/peckhamcc/www/blob/master/packages/config/index.js#L619), then accessed [via env vars in production mode](https://github.com/peckhamcc/www/blob/master/packages/config/index.js#L664) and [set at build time](https://github.com/peckhamcc/www/blob/master/packages/website/esbuild.js#L39) for the front end.

Locally your dev setup will use the sandbox value, the lambdas will use the production env var and the deployed frontend will use the build time value all from the same source file.

File: packages/website/src/app.jsx (hunk 1 above)
Comment (achingbrain) [reply packages/website/src/app.jsx:137]: Please do, we can bring them back via the git history if necessary.

We should also be able to delete the content pages these routes linked to as well now, right?

File: packages/website/src/components/contentful-page/helpers/page.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { setupClient } from '../../../shared/contentful'
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer'
+
+export const getPageFromId = async (entryId) => {
+  const client = setupClient()
```
Comment (achingbrain) [inline packages/website/src/components/contentful-page/helpers/page.jsx:5]: We seem to set up a new client for every interaction with contentful - would it be better to have one for the whole app passed down through props instead?

**Review by achingbrain — CHANGES_REQUESTED** [review]
LGTM, a few comments inline.

**Comment by achingbrain** [comment]
How do people go about creating news pages with contentful, or is that coming in a future PR?

File: packages/website/src/app.jsx (hunk 1 above)
Comment (owenmerry) [reply packages/website/src/app.jsx:137]: yep will clean these and delete the files too when doing the PR updates👍

File: packages/website/src/shared/contentful.js (hunk 2 above)
Comment (owenmerry) [reply packages/website/src/shared/contentful.js:5]: ah cool, we have configs.. let me have a look how these work and move the contentful ones over to there

**Comment by owenmerry** [comment]
> This is awesome, btw.
> 
> Is there a plan for how people go about creating news pages with contentful?

Thanks 😁... i didnt think about news articles.. but i would create them as a different contentType and have a page which shows them all.. that way in contentful for editors all they have to do is create a news page.. we can then add in specific news features like date and author or something.. i can do this part of a different PR.. if this goes out you can create a simple setup using the pages feature i have now and then i will just migrate over the news pages to be news content types

**Comment by owenmerry** [comment]
@achingbrain all changes done expect the passing the client in props, if you still feel we need that 

i created a development apis and a production one we can use for the site
view here https://app.contentful.com/spaces/6dcasuqyhf2a/api/keys
Screenshot 2022-03-07 at 21 39 01

**Review by achingbrain — APPROVED** [review]

**Comment by achingbrain** [comment]
You could wrap each component that needs the contentful client in a HOC that provides a global instance, no need to pass it down through the app.

Anyway that can be done in the future.

**Pull request merged**